packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -7,85 +7,94 @@
     >
       <oc-loader v-if="linksLoading" :aria-label="$gettext('Loading list of file links')" />
       <template v-else>
-        <h3 class="oc-text-bold oc-m-rm oc-text-initial" v-text="linksHeading" />
-        <div v-if="canCreatePublicLinks" class="oc-my-s">
-          <p class="oc-text-muted">
-            <span v-translate
-              >Any external person with the respective link can access this resource. No sign-in
-              required. Assign a password to avoid unintended document exposure.</span
-            >
-            <oc-contextual-helper v-if="helpersEnabled" v-bind="viaLinkHelp" />
-          </p>
+        <h3 class="oc-text-bold oc-m-rm oc-text-initial">
+          <span v-text="linksHeading" />
+          <oc-contextual-helper v-if="helpersEnabled" v-bind="viaLinkHelp" />
+        </h3>
+        <div v-if="canCreatePublicLinks" class="oc-mt-m">
+          <!-- quicklink goes here -->
+          <!-- <hr class="oc-my-s"> -->
```

Comment:
leftover?

Reply:
no, reminder for future implementation (quicklink)

Reply:
To be cleaned up in the near future in another PR